LSpec/Config.lean

```diff
@@ -0,0 +1,3 @@
+-- TODO: Config data structure more complicated than this
+structure Config where
+  temp : Bool
```

Comment:
Do we need this? Configuring the behavior of an imported package should be done with a proper option via `set_option`, I believe. So this probably wouldn't be the way

Reply:
This is just a placeholder, so you can delete it for now. I had it in place to eventually support some of the configurations like they have here

https://hspec.github.io/options.html

If you think set_option can subsume this kind of functionality then we won't need it.